ynh_mongo_db/ynh_mongo_db

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Create a Database in mongodb
+# Usage : ynh_mongo_setup_db dbname dbuser
+ynh_mongo_setup_db() {
+  local db_name=$1
+  local db_user=$2
+  local db_pass=$(ynh_string_random)
+  # Create a user for the Database
+  mongo --shell "$db_name" --eval 'db.createUser( { user: "'${db_user}'", pwd: "'${db_pass}'", roles: [ "readWrite" ] } );' <<< exit
+  # Set the user as admin of the db
+  mongo --shell "$db_name" --eval 'db.grantRolesToUser("'${db_user}'",[{ role: "clusterMonitor", db: "admin" }]);' <<< exit
+}
+
+# Remove a Database in Mongodb
+# Usage : ynh_mongo_remove_db dbname
+ynh_mongo_remove_db() {
+  local db_user="$1"
+  local db_name="$2"
+  mongo --shell "$dbname" --eval 'db.dropUser("'${dbuser}'", {w: "majority", wtimeout: 5000})' <<< exit
+  mongo --shell "$dbname" --eval 'db.runCommand({dropDatabase: 1})' <<< exit
+}
+
+ynh_mongo_dump_db() {
+  local db_name=$1
+  date_today=$(date +"%m-%d-%y")
+  mkdir -p /var/backups/mongobackups/
+  find /var/backups/mongobackups/ -mtime +7 -exec rm -rf {} \;
+  mongodump --db $db_name --out /var/backups/mongobackups/$date_today
```

Comment:
Isn't there a way to dump the db to stdout to match the behavior of mysql and postgresl helper ?

Reply:
Seems like `mongodump --archive` can do the job
https://docs.mongodb.com/manual/reference/program/mongodump/

```
--archive=<file>

 New in version 3.2.

Writes the output to a specified archive file or, if the archive file is unspecified, writes to the standard output (stdout).
```

Will try to do that on #64